pkg/tagger/collectors/ecs_main.go

```diff
@@ -9,24 +9,32 @@ package collectors
 
 import (
 	"fmt"
+	taggerutil "github.com/DataDog/datadog-agent/pkg/tagger/utils"
 	ecsutil "github.com/DataDog/datadog-agent/pkg/util/ecs"
+	"time"
 )
 
 const (
 	ecsCollectorName = "ecs"
+	ecsExpireFreq    = 5 * time.Minute
 )
 
 // ECSCollector listen to the ECS agent to get ECS metadata.
 // And feed a stream of TagInfo.
```

Comment:
Please update, we're not streaming. Also, say it relies on the DockerCollector to trigger deletions. That'll make clear it's not intended to run standalone. These collector class comments could benefit from being longer, don't hesitate to add contents about intents, performance...

Also, update the `README.md` with the fact ECSCollector does not run in pull mode, and the reason why

Reply:
It took me a few seconds to realize that the comment is on the comment, not on the declaration of ecsExpireFreq. Indeed Stream here is not adapted, I meant channel but I understand point.
👌